Examples/UseCases.ipynb

```diff
@@ -129,7 +144,16 @@
    "source": [
     "from pydkpro import DKProCoreTypeSystem as dts\n",
     "\n",
-    "cas.select(dts().token).as_text() "
+    "cas.select(dts().token).as_text() \n",
+    "\n",
+    "// REC: I wonder which version of the DKProCoreTypeSystem is used here...\n",
```

Comment:
That is good point, here ```dts()``` is DKPro type system class object call which load DKPro Core type systems (similar to **DKPro Core Integration** in ```dkpro-cassis``` wrapper) with latest version and return as CAS object. I believe we should use it to automatically load the type systems. Version of typesystem can also passed while calling this object

```
from pydkpro import DKProCoreTypeSystem as dts

ts_token = dts(version="2.0.0").token

cas.select(ts_token).as_text() 
```

Please provide your feedback, if you like or dislike above way of representation.

Reply:
If it is an object, it should start with a Capital letter, no?

If it is a utility method, maybe `load_dkpro_core_typesystem(version="2.0.0")` would make it clearer.

Personally, I would find `load_dkpro_core_typesystem(version="2.0.0").get_token_type()` quite understandable.

```
ts = load_dkpro_core_typesystem(version="2.0.0");
cas.select(ts.get_token_type()).as_text();
```

or maybe

```
ts = load_dkpro_core_typesystem(version="2.0.0");
cas.select(ts.token_type).as_text();
```

Reply:
Yes, it looks nicer. But here with ```load_dkpro_core_typesystem``` you mean replacing it with ```DKProCoreTypeSystem``` class name and not just use it as alias name.  If yes, I'll update notebook like following:

```
ts = Load_dkpro_core_typesystem(version="2.0.0")
cas.select(ts.token_type).as_text()
```

Reply:
For me `Load_dkpro_core_typesystem(...)` would be a utility method, not a constructor - so `load_dkpro_core_typesystem(...)`. If you wanted to use a class, I would think of something like `DKProCoreTypesystemLoader().load(version="2.0.0")` but that seems a bit clumsy to me - why use a class here if a utility method suffices?